SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.cc

```diff
@@ -699,7 +704,7 @@ void Phase2TrackerDigitizerAlgorithm::pixel_inefficiency(const SubdetEfficiencie
 
   // setup the chip indices conversion
   unsigned int Subid=DetId(detID).subdetId();
-  if (Subid == PixelSubdetector::PixelBarrel) { // barrel layers
+  if (Subid == PixelSubdetector::PixelBarrel || Subid == StripSubdetector::TOB) { // barrel layers
```

Comment:
@davidlange6  - here TOB means the phase2 barrel layers as defined in the package description of Geometry/TrackerNumberingBuilder in the section of 

"Phase 2 Upgrade Detector DetId schema Tilted Geometry with new pixel detector for TDR (from Version V4021)"

https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Geometry/TrackerNumberingBuilder/README.md